src/main/java/org/candlepin/subscriptions/http/HttpClientProperties.java

```diff
@@ -21,8 +21,10 @@
 package org.candlepin.subscriptions.http;
 
 import java.io.File;
+import lombok.Data;
 
 /** HTTP service client configuration. */
+@Data
```

Comment:
Consistent with previous work, this should be Getter and Setter, since those are the only methods you're replacing.  The aspect of Data that we're avoiding is RequiredArgsConstructor.

Reply:
Done (see latest commit)